homeassistant/components/amberelectric/translations/en.json

```diff
@@ -0,0 +1,27 @@
+{
+    "title": "amberelectric",
+    "config": {
+        "step": {
+            "user": {
+                "data": {
+                    "api_token": "API Token",
+                    "site_id": "Site ID",
+                    "site_nmi": "Site NMI",
+                    "site_name": "Site Name"
+                },
+                "title": "Amber Electric",
+                "description": "Go to https://app.amber.com.au/developers to generate an API key"
```

Comment:
Don't put URLs in translatable strings. Instead inject it via a placeholder.

Reply:
I'm not 100% sure how to do this? Have you got an example?

Reply:
All good - worked it out